README.md

```diff
@@ -87,21 +87,23 @@ doctor ordered.
 
 ## Install as a cron
 
-If you want to get an email every night to give you a list of all the soft
-constraints that have been broken during the last day, just add it to you
-crontab. We like to have it run daily, so we can fix any bug generating bad
-data before it becomes a real problem.
+Run sdic from cron as often as you like, we like to have it run daily.
 
-Example crontab:
+## Monitoring
 
-```
-MAILTO="dba@acme.com"
-@daily sdic live
+sdic uses Python's [`logging`](https://docs.python.org/2.7/library/logging.html)
+module to log any output in dot separated hierarchical fashion.
+
+Any general sdic message would look like this:
+
+```console
+Jan 11 00:10:19 <sender> sdic.enforce_fullname: enforce_fullname.sql successfully ran in 0.029 sec
 ```
 
-`dba@acme.com` is the email that will get the soft constraints broken every
-day. Make sure your local MTA is well configured on your system. You can test
-it by doing `date | mail -s test dba@acme.com`.
+In Papertrail `sdic.enforce_fullname` will be treated as a
```

Comment:
Papertrail is really specific to us, can you talk about syslog in general instead?